From bf4ed7724eca409535b40d69180b07d5da3f378d Mon Sep 17 00:00:00 2001
From: Alan Stern <stern@rowland.harvard.edu>
Date: Mon, 7 May 2012 13:22:52 -0400
Subject: [PATCH 11/50] USB: xhci-hcd: print URB's expected length in decimal,
 not hex
Git-Commit: 1949f9e2435cc2e87c63da167af63faafc49b388
Patch-Mainline: v3.5
References: bnc#920620

This patch changes the output format specifier of a debugging line in
the xhci-hcd driver.  An URB's transfer_buffer_length should be
printed in decimal; there's no reason to print it in hex.  Especially
since the actual_length value, printed earlier on the same line, is
already in decimal.

Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
Cc: Sarah Sharp <sarah.a.sharp@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Oliver Neukum <oneukum@suse.de>
---
 drivers/usb/host/xhci-ring.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/xhci-ring.c b/drivers/usb/host/xhci-ring.c
index 867524e..47ee2293 100644
--- a/drivers/usb/host/xhci-ring.c
+++ b/drivers/usb/host/xhci-ring.c
@@ -2606,7 +2606,7 @@ cleanup:
 					(status != 0 &&
 					 !usb_endpoint_xfer_isoc(&urb->ep->desc)))
 				xhci_dbg(xhci, "Giveback URB %p, len = %d, "
-						"expected = %x, status = %d\n",
+						"expected = %d, status = %d\n",
 						urb, urb->actual_length,
 						urb->transfer_buffer_length,
 						status);
-- 
1.8.4.5

